Makefile

```diff
@@ -91,19 +94,22 @@ build-backend: ## Builds all backend containers
 	@echo "==============================================="
 	@echo "Make: build-backend - building backend images"
 	@echo "==============================================="
-	@docker-compose -f docker-compose.yml build db db_setup api nginx clamav
+	@docker-compose -f docker-compose.yml build db db_setup api nginx
+## @docker-compose -f docker-compose.yml build db db_setup api nginx clamav
```

Comment:
can this commented line be removed?

Reply:
same question for few more places below

Reply:
I'm just commenting out the clamav step, but didnt want to lose track of it as we will eventually set that up